CONTRIBUTING.md

```diff
@@ -8,4 +8,24 @@ This document builds on the [default contributing.md](https://github.com/comcode
 
 ## How to make changes to code
 
+### Creating tests
+
+1. For some module `foo.ts`, create a file `foo.test.ts` in the same directory. (Must be within /src)
```

Comment:
Small thing, but I'd give our example using `.js` to reach as many people as possible. Those who don't know TS won't be intimidated by the instructions, and those who do will likely know enough to change the extension for their needs.

Reply:
@danswann I agree that these instructions need to be refined. For this PR I wanted to specify what is allowed by our config, so I have now listed the other extensions, but I think further examples are more appropriate as separate issues.

I imagine future test instructions would be agnostic to the module type, providing guidance on jest itself, and referencing the 'changes to code' section as needed for import syntax and such.

Reply:
Yeah, sorry, I just meant changing `foo.ts` and `foo.test.ts` to `foo.js` and `foo.test.js`, rather than any kind of refinement. Just to avoid people being instantly intimidated and thinking TS is a prereq to testing.